docker-compose-sgx.yaml

```diff
@@ -39,12 +39,12 @@ services:
     entrypoint: bash -c "tail -f /dev/null"
     stop_signal: SIGKILL
     depends_on:
-      - avalon-sgx-enclave-manager
+      - avalon-enclave-manager
       - avalon-listener
 
-  avalon-sgx-enclave-manager:
-    container_name: avalon-sgx-enclave-manager
-    image: avalon-sgx-enclave-manager-dev
+  avalon-enclave-manager:
```

Comment:
The reason we named the service this way is to differentiate between enclave manager running in SIM mode and HW mode. In the listener config file, by default we can config zmq_url to point to sim mode enclave manager service.
You can mention in a comment that in case of HW to use HW mode enclave manager service.

Reply:
We are sharing dockerfile for both SIM and HW mode then, what are we differentiating between these 2 modes? Are we supporting to run both sim and HW modes at a time in a machine?

Reply:
Yes, we currently use same dockerfile for both SIM and HW mode. Differentiating factor is SGX_MODE arg that is passed from compose file. Also, in case of HW mode, we map the device from compose file. Its possible to run both SIM and HW modes on SGX enabled platform but not at a time.

Reply:
If both SIM and HW modes are not running at time then why do we need to need to differentiate with docker service name. I agree SGX_MODE env variable and device mount 
 needs to passed in case of hardware mode, but these run time arguments to docker service to run(volume and ENV variable). I don't any issue in using same docker service name for both SIM and HW mode.

Reply:
In the same system we can run both SGX SIM and HW mode. So we need to have different docker image name for enclave manager one for SGX SIM(avalon-enclave-manager-dev) and SGX HW(avalon-sgx-enclave-manager-dev) .

Reply:
@manojgop Docker images for both SGX SIM and HW mode is same because  Dockerfile is same for both modes. All we need run time arguments for HW mode ENV and volume mounting, that is the only difference.

Reply:
I changed i to same docker service name in both SIM and HW docker file, it is working for me.

Reply:
Dockerfile is same. But docker image generated is different for SGX HW and SIM. The same image won't work for SGX HW and SIM.  check https://github.com/hyperledger/avalon/blob/master/enclave_manager/Dockerfile#L211

Reply:
For configuration in `listener/listener_config.toml` and documentation in `docs/FAQ.rst` it is easier if we have the same hostname for both Intel SGX SIM and HW modes... that is `tcp://avalon-enclave-manager:5555`

Reply:
@danintel @Ram-srini  We can probably keep the service name same , but let's keep different docker image name for SGX HW and SIM. If we have same docker image name then on same system it's not easier to test both since if we build image for SGX-SIM and then for SGX-HW then image will be overwritten.

Reply:
@manju956 @manojgop I have changed docker image name as suggested, review it.